services/director-v2/tests/integration/02/test_dynamic_services_routes.py

```diff
@@ -117,6 +125,7 @@ async def test_client(
     monkeypatch.setenv("SIMCORE_SERVICES_NETWORK_NAME", network_name)
     monkeypatch.delenv("DYNAMIC_SIDECAR_MOUNT_PATH_DEV", raising=False)
     monkeypatch.setenv("DIRECTOR_V2_DYNAMIC_SCHEDULER_ENABLED", "true")
+    monkeypatch.setenv("DYNAMIC_SIDECAR_LOG_LEVEL", "DEBUG")
```

Comment:
minor and just a thought... the other day I spent a few minutes here wondering why this ```test_``` fct was never called ;)... could we agree to not call fixtures test_XXX ?

Reply:
Yes. Let me drop this already. Would be quite annoying for people.